tapir/accounts/tests/test_ldap_groups_management.py

```diff
@@ -132,3 +133,16 @@ def test_member_office_cannot_access_group_list_view(self):
 
         response = self.client.get(reverse("accounts:ldap_group_list"))
         self.assertEqual(403, response.status_code)
+
+    def test_editUserLdapGroupsView_changingUserGroupsCreatesLogEntry(self):
+        self.login_as_vorstand()
+        tapir_user = TapirUserFactory.create(is_in_vorstand=False)
+        data = {GROUP_VORSTAND: True}
+        response = self.client.post(
+            reverse("accounts:edit_user_ldap_groups", args=[tapir_user.pk]),
+            data,
+            follow=True,
+        )
+        self.assertStatusCode(response, HTTPStatus.OK)
+        self.assertEqual(1, UpdateTapirUserLogEntry.objects.count())
+        self.assertTrue(GROUP_VORSTAND in tapir_user.get_ldap_user().group_names)
```

Comment:
```suggestion
        self.assertIn(GROUP_VORSTAND, tapir_user.get_ldap_user().group_names)
```
We can get slightly better messages if the test fails with `assertIn`